src/commands/project/retrieve/start.ts

```diff
@@ -183,7 +183,9 @@ export default class RetrieveMetadata extends SfCommand<RetrieveResultJson> {
     const retrieveOpts = await buildRetrieveOptions(flags, format, zipFileName, resolvedTargetDir);
 
     this.spinner.status = messages.getMessage('spinner.sending', [
-      componentSetFromNonDeletes.sourceApiVersion ?? componentSetFromNonDeletes.apiVersion,
+      componentSetFromNonDeletes.sourceApiVersion ??
+        componentSetFromNonDeletes.apiVersion ??
+        flags['target-org'].getConnection().getApiVersion(), // eslint-disable-line sf-plugin/get-connection-with-version
```

Comment:
I think we should just remove the apiVersion from the spinner message because:

1. The plugin can display the wrong apiVersion.  If SDR is being used (and it is most of the time) it should be the source of truth.  E.g., the command `sf project retrieve start --api-version=68.0` will cause apiVersion 68.0 to display in the spinner message when in fact version 58.0 was used by SDR.
2. The `apiVersionRetrieve` Lifecycle event handled above should display the correct apiVersion that was used by SDR.  If both messages output they can display conflicting versions.  SDR is the only reliable place that apiVersion is resolved.